src/w3c/templates/headers.js

```diff
@@ -2,16 +2,50 @@ import "deps/hyperhtml";
 import showLogo from "./show-logo";
 import showPeople from "./show-people";
 import showLink from "./show-link";
+import { pub } from "core/pubsubhub";
 
 export default conf => {
   const html = hyperHTML;
+  
+  function getSpecTitleElem () {
```

Comment:
These should be moved outside this function.

Reply:
I had initially kept these 2 functions outside the `export` statement only. However, `conf` would then be undefined and thus, 5 out of 6 tests fail :disappointed: . I guess we'll have to keep them inside `export` only

Reply:
Ah, you can just pass conf in:

```js
function getSpecTitleElem(conf) {

}
```

Reply:
how can i miss that. So sorry

Reply:
Heh, it happens to all of us :) why reviews matter.